sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Models/TimeSeriesPoint.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// Represents a Time Series point at a specific timestamp.
```

Comment:
Does a point here mean timestamp and property values?

Reply:
Yes. You can think of each point as a point on the TSI graph

Reply:
We can use feedback of team here for better suggestions. To me it seems describing it a little bit more would be helpful.